app/src/main/kotlin/com/github/oryanmat/trellowidget/util/color/ColorPreference.kt

```diff
@@ -23,6 +23,15 @@ class ColorPreference @JvmOverloads constructor(
         dialogLayoutResource = R.layout.color_chooser
     }
 
+    fun repairSVBar(view: View, colorToFix: Int) {
+        // This is a work-around for a defect in holocolorpicker when using black with an SVBar
```

Comment:
I will try to submit a pull request to that repo as well, but figure this workaround is pretty lightweight until (if) that fix goes in there.